### ISSUE TITLE
Disabling second factor does not ask for confirmation

### ISSUE THREAD

**Issue by morph027** [nextcloud/twofactor_gateway#97]
Just tried the thing using Signal and it's awsome, just works straight away.

Just a hint, disabling the second factor should probably ask for one last confirmation code ;)

**Comment by ChristophWurst** [comment]
Hey, I'm glad you like it, esp. since it's your Docker image that we use to send the codes! Are you at the conf? ✌

Yes, good point. We have to add confirmation. Other 2FA prividers have also password protection for enabling/disabling the provider. Would make sense to add this here, too.

**Comment by morph027** [comment]
Yes, was at the conf but need to leave now for the bus. Would have loved to talk more about if only I had knew before.

Anyway, I'm pretty sure we can solve things here too ;)

**Comment by ChristophWurst** [comment]
What a pity!

Yes, we can :rocket:
